Dockerfile

```diff
@@ -96,30 +95,38 @@ FROM acousticbrainz-base AS acousticbrainz-prod
 
 ARG deploy_env
 
-RUN pip install uWSGI==2.0.17.1
+RUN pip install --no-cache-dir uWSGI==2.0.17.1
+
+RUN mkdir /cache_namespaces
+RUN chown -R www-data:www-data /cache_namespaces
```

Comment:
Where are www-data user/group created ?

Reply:
I don't know. They've always been here (we've been running uwsgi as www-data since we deployed it)